README.md

```diff
@@ -1,12 +1,48 @@
-<h1 align="center">Square Snippet Demo - swif.club</h1>
+# SWIF - Social Shopping app powered by Square Unboxed - API snippet [![CircleCI](https://circleci.com/gh/failure-driven/square-snippet-demo.svg?style=svg)](https://circleci.com/gh/failure-driven/square-snippet-demo)
+## Enables your Square powered app to add powerful "Interactive Social Shopping Feature" in 60 seconds
 
-<div align="center">
+[![Square Unboxed - API snippet for Rails Template Walkthrough](https://i.ibb.co/6X4wXtw/Screen-Shot-2021-06-08-at-2-28-36-pm.png)](https://www.youtube.com/watch?v=JbytzNibk54)
 
-[![CircleCI](https://circleci.com/gh/failure-driven/square-snippet-demo.svg?style=svg)](https://circleci.com/gh/failure-driven/square-snippet-demo)
+## Requirements
 
-</div>
+You'll need the following installed to run the template successfully:
 
+* Ruby 3.0.1 or higher
+* bundler - `gem install bundler`
+* rails - `gem install rails`
+* Yarn - `brew install yarn` or [Install Yarn](https://yarnpkg.com/en/docs/install)
+
+Other:
+* Svelte Compiler/ JS framework (Provided out of the box) - lightweight and fast alternative to React/Vue/Angular
+* Database (MySQL Provided out of the box)
```

Comment:
does it work with mysql? we have postgres and the assoicated gems so it should work with postgresql. Again I am not sure how to ensure this. in general I use `.tool-versions` which is part of asdf to manage all of that (you should too -> https://asdf-vm.com/#/core-manage-asdf) maybe we just mandate that to work on this project and get all dependencies right you need asdf